### PR TITLE
Add ADR about design system change

### LINKED ISSUE #889: Document rationale and decision about design system replacement
The [refactor/architecture ADR](https://github.com/NASA-IMPACT/veda-ui/pull/875) includes notes about a change of design system. This should be documented in a separate ADR.

## Acceptance criteria

- [ ] Wrote new ADR for design system

### PR DESCRIPTION
**Pull request by j08lue**
Closes #889 

### Description of Changes

Adds ADR about design system change

### Notes & Questions About Changes

Based on content previously included in refactor ADR #875 

### Validation / Testing

https://github.com/NASA-IMPACT/veda-ui/blob/adr/design-system-change/docs/adr/003-design-system-change.md

### CODE REVIEW

File: docs/adr/003-design-system-change.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+✔️ [2] Introduce new design system
+
+We decided to Replace the existing custom design system with a well-maintained, widely adopted community-supported design system. This ensures built-in accessibility standards and eliminates the need for designers and developers to invest time in ongoing maintenance.
+
+The preliminary systems we are considering are:
+  * [Chakra UI](https://chakra-ui.com/). Built on React, it has a large component library, built-in accessibility and theming options. It is designed with responsive and mobile-first principles and has a strong community and documentation.
+  * The [U.S. Web Design System (USWDS)](https://github.com/uswds/uswds) is tailored for government projects. It's designed to meet the specific needs of U.S. federal websites, offering components that are accessible, secure, and in line with U.S. digital standards.
+
+Any option we choose will have to be extended for data visualization components (charts, maps, widgets, etc.) and themed for the 3 current VEDA instances: VEDA Dashboard, U.S. GHG Center, Earth.gov.
+
```
Comment (sandrahoang686) [inline docs/adr/003-design-system-change.md:47]: Should we note that which design system is still "to be determined"? So it leaves space and serves as sort of a reminder to come back to update this ADR?

Comment (j08lue) [inline docs/adr/003-design-system-change.md:47]:
```suggestion

The final choice for a web design system is not subject of this ADR and will be documented separately.
```

**Comment by j08lue** [comment]
@sandrahoang686 @hanbyul-here @faustoperez - I think this ADR is not controversial and accurately documents the decision made. Should we push it over the finish line?

**Review by AliceR — APPROVED** [review]

**Comment by j08lue** [comment]
Think this is trivial, so I'll go ahead and merge it. Thanks @AliceR.

**Pull request merged**